### PR TITLE
[WIP] CLI and CMD refactor

### PR DESCRIPTION
**Pull request by GregTheGreek**
This pr introduces a CLI and re-structures the project into more useable components.

It also has chainsafe imports because of some local development i was tinkering with..

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -35,13 +35,13 @@ $ bitcoin-cli stop
 * get utreexo code
 
 ```
-$ go get github.com/mit-dci/utreexo
+$ go get github.com/chainsafe/utreexo
```
Comment (kcalvinalvin) [inline readme.md]: Should change all of these back to mit-dci for a PR

Comment (GregTheGreek) [reply readme.md]: addressed

File: node/main.go [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package node
+
+import (
+	log "github.com/ChainSafe/log15"
```
Comment (kcalvinalvin) [inline node/main.go]: Wouldn't it be better if the changes were pushed upstream and that was imported?

**Comment by kcalvinalvin** [comment]
Haven't looked through everything but seems like the main difference is that you've already added the ability to add a config file along with logging.

File: node/main.go (hunk 2 above)
Comment (GregTheGreek) [reply node/main.go]: Not sure im following you? 

This is a custom package, just a simple logger.

Comment (kcalvinalvin) [reply node/main.go]: hmm never mind, the original repo that you guys forked off of seems abandoned.

Comment (GregTheGreek) [reply node/main.go]: ohhh, yeah sorry if that wasn't clear. We also updated it a bit.

**Pull request closed without merging**